pages/login.vue

```diff
@@ -0,0 +1,296 @@
+<template>
+  <div class="login-page">
+    <v-btn
+      rounded
+      fab
+      small
+      icon
+      :ripple="false"
+      class="nav-back-btn"
+      @click="to"
```

Comment:
`to` is a bad name for a method, there must be some action `goToBack` for example

Reply:
agree

Reply:
my bad, didnt cleanup properly